### PR TITLE
Add cleanup 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- The code quality including spaces could be better. #24 
- There is a `scrollToTop` function which is not being used. #25 

**What is the new behavior (if this is a feature change)?**
- The code quality has been improved. #24 
- The `scrollToTop` function has been removed. #25 

**Additional**
- The version has been updated to 1.0.2

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks okay to me.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-mdb-template",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (kleberbaum) [inline package.json:3]: This update isn't mentioned in the PR.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: package.json (hunk 1 above)
Comment (Aichnerc) [reply package.json:3]: Done

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**